src/storage.rs

```diff
@@ -68,16 +71,34 @@ impl Default for AccountBasic {
     }
 }
 
-/// EVM Code, currently mapping to REVM's [ByteCode::LegacyAnalyzed].
-// TODO: Support raw legacy & EOF
+/// Analyzed legacy code.
 #[derive(Debug, Clone, PartialEq, Default, Serialize, Deserialize)]
-pub struct EvmCode {
-    /// Bytecode with 32 zero bytes padding
-    bytecode: Bytes,
-    /// Original bytes length
-    original_len: usize,
+pub struct LegacyCode {
+    /// Bytecode with 32 zero bytes padding.
```

Comment:
The `Eip7702Code`'s raw bytecode conversation made me realize we don't need to store padded bytecodes as well. Perhaps add a `// TODO: Store unpadded bytecode and pad on revm conversion` here?